package.json

```
@@ -15,18 +11,60 @@
   "bugs": {
     "url": "https://github.com/onCondition/onCondition-client/issues"
   },
-  "homepage": "https://github.com/onCondition/onCondition-client#readme",
+  "private": true,
+  "dependencies": {
+    "@reduxjs/toolkit": "^1.6.1",
+    "firebase": "^9.0.1",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-redux": "^7.2.4",
+    "react-redux-firebase": "^3.10.0",
+    "react-router-dom": "^5.2.1",
+    "react-scripts": "4.0.3",
+    "redux-logger": "^3.0.6",
+    "styled-components": "^5.3.1"
+  },
   "devDependencies": {
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^12.8.3",
     "eslint": "^7.30.0",
+    "eslint-plugin-react": "^7.25.1",
     "husky": "^4.3.8",
+    "lint-staged": "^11.1.2",
     "stylelint": "^13.13.1",
     "stylelint-config-recommended": "^5.0.0"
   },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
   "husky": {
     "hooks": {
-      "pre-push": "git branch | grep \"*\" | egrep -v \"^\\* main$\""
+      "pre-push": "git branch | grep \"*\" | egrep -v \"^\\* main$\"",
+      "pre-commit": "lint-staged"
     }
   },
+  "lint-staged": {
+    "*.js": [
```

```suggestion
    "*.{js, jsx}": [
```
문법이 정확한진 모르겠지만 jsx로 컴포넌트 사용하실거라면 jsx도 추가해야 할거 같아요.

---

확인 결과 말씀해주신 문법이 맞아서 반영했습니다!